### PR TITLE
Nft metadata

### PR DESCRIPTION
**Pull request by motechFR**

### CODE REVIEW

File: apps/scoutgamecron/src/scripts/fixArtwork.ts [hunk 1]
```diff
@@ -12,7 +13,7 @@ async function refreshArtworks() {
   const builderNfts = await prisma.builderNft.findMany({
     where: {
       season: currentSeason,
-      contractAddress: builderContractAddress
+      contractAddress: getBuilderContractAddress().toLowerCase()
```
Comment (mattcasey) [inline apps/scoutgamecron/src/scripts/fixArtwork.ts:16]: Why do we need to lowercase this? the only reason we lowercased walelt addresses in webapp is to normalize them, honestly not sure we should have done that now looking back

File: .ebstalk.apps.env/scoutgame.env [hunk 2]
```diff
@@ -28,4 +28,5 @@ GITHUB_ACCESS_TOKEN="{{pull:secretsmanager:/io.cv.app/prd/github:SecretString:sc
 
 BUILDER_SMART_CONTRACT_OWNER_PRIVKEY="{{pull:secretsmanager:/io.cv.app/prd/credentials:SecretString:builder_smart_contract_owner_privkey}}"
 
-REACT_APP_ENABLE_BUILDER_SMART_CONTRACT="true"
\ No newline at end of file
+NFT_ARTWORK_S3_PATH="prd-artworks"
```
Comment (mattcasey) [inline .ebstalk.apps.env/scoutgame.env:31]: "prd" should be enough? its not just artworks, we also put metadata in this folder for each token

File: packages/scoutgame/src/builderNfts/constants.ts [hunk 3]
```diff
@@ -27,17 +25,9 @@ const devOptimismMainnetBuildersContract = '0x1d305a06cb9dbdc32e08c3d230889acb9f
...
-  ? realOptimismSepoliaBuildersContract
-  : devOptimismSepoliaBuildersContract;
-const optimismMainnetBuilderContractAddress = useRealContract
-  ? realOptimismMainnetBuildersContract
-  : devOptimismMainnetBuildersContract;
-
-export const builderContractAddress = useTestnets
-  ? optimismSepoliaBuilderContractAddress
-  : optimismMainnetBuilderContractAddress;
+export function getBuilderContractAddress(): `0x${string}` {
```
Comment (mattcasey) [inline packages/scoutgame/src/builderNfts/constants.ts:28]: why is this a getter? i dont think u need to do that

**Review by mattcasey — APPROVED** [review]
left couple comments but code looks ok. thanks for adding the switch wallet to nft purchase

File: packages/scoutgame/src/builderNfts/constants.ts (hunk 3 above)
Comment (motechFR) [reply packages/scoutgame/src/builderNfts/constants.ts:28]: Make sure env() evaluates to something as react-env may not be loaded yet

File: .ebstalk.apps.env/scoutgame.env (hunk 2 above)
Comment (motechFR) [reply .ebstalk.apps.env/scoutgame.env]: Fixed & also copied files to new destination

**Pull request merged**